### DISCUSSION TITLE
Dropshadow in filter CSS property is not working in Tauri on Linux

### DISCUSSION THREAD

**Discussion by aagahkanat** [tauri-apps/tauri#8808] [Q&A]
The filter I use in an svg animation: drop-shadow(..)..; It doesn't work on ubuntu. I tried both filter and -webkit-filter and tried many suggested solutions I found. However, I could not solve this problem.

**Comment by FabianLars** [comment]
I assume you tested this in Firefox or a chrome like browser to make sure your code is correct, right?

If so, could you please test it in epiphany (also known as gnome web) to check if it's a general webkitgtk problem?

**Reply by aagahkanat** [reply]
When I tested it with both -webkit-filter and filter in Gnome, it didn't work either.

**Reply by FabianLars** [reply]
That is unfortunate. It means it's a webkitgtk issue which we can't fix.

I don't see something relevant [here](https://bugs.webkit.org/buglist.cgi?quicksearch=drop-shadow), do you? If not, we'd appreciate it if you could open a bug report there :)

**Reply by aagahkanat** [reply]
Thanks for all your answers. Many bugs related to drop-shadow have been added to the list. Tauri is still using webkit2gtk 0.18 released 2 years ago. Maybe an update on this issue will solve such problems.

**Reply by FabianLars** [reply]
This is not the engine version, just the version for the bindings crate and we make up the version for that ourselves.

The engine being used is whatever is on the system. And if you installed epiphany via flatpak it will always be the latest even if your distro didn't ship it yet.
